src/js/app.js

```diff
@@ -386,4 +395,5 @@ $(() => {
   checkForActionParameter()
   tryToRemoveNewsletter()
   trySearchFeature()
+    openExternalLinksInDifferentTab()
```

Comment:
@rubalv96 could you please indent this as the others 🙈

Reply:
upsss 🙊
solved!